### PR TITLE
ToDo list review for JS best practice checks

### PR DESCRIPTION
**Pull request by charlyeneh**
Highlights:-
- A copy of the ToDo list app was created on the feature branch.
- A request for review will be done from my partner.

### CODE REVIEW

File: dist/bundle.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+/******/ 	
+/************************************************************************/
+/******/ 	
+/******/ 	// startup
+/******/ 	// Load entry module and return exports
+/******/ 	// This entry module can't be inlined because the eval devtool is used.
+/******/ 	var __webpack_exports__ = __webpack_require__("./src/index.js");
+/******/ 	
+/******/ })()
+;
```
Comment (estebanmual) [inline dist/bundle.js:238]: I would recommend deleting all the comments that appear in the file as it is unnecessary code. You can do this automatically through the optimization of the file within the webpack config file.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+			integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+			crossorigin="anonymous"
+		/>
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
```
Comment (estebanmual) [inline src/index.html:13]: A good practice to follow is that in the template file we declare in webpack, we should not include any link tag or script tag. My suggestion is to include them through webpack process.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    </header>
+  <div class="main-panel">
+    <input id="add-input" class="add-input" type="text" placeholder="Add to your list">
+    <ul id="todo-list">
+
+    </ul>
+    <button id="clear-btn" class="clear-btn" type="button" name="button">Clear all completed</button>
+  </div>
+</main>
+<script type ="module" src="index.js"></script>
```
Comment (estebanmual) [inline src/index.html:30]: A good practice to follow is that in the template file we declare in webpack, we should not include any link tag or script tag. My suggestion is to include them through webpack process.

**Review by estebanmual — CHANGES_REQUESTED** [review]
Hi Charles, great job doing the activity :fire: 
I loved the design and functionalities you included :100: 

I only have some little suggestions that you can check through the code review.

**Comment by charlyeneh** [comment]
Hi @estebanmual,  
Thank you for your kind reviews.

A look at the above issues raised looks like they are optional changes. I will look into them and update them where necessary.

Best regards and happy coding.

**Review by estebanmual — APPROVED** [review]
Great job Charles! Nothing else to add! 💯 🔥

**Pull request merged**